pyvcloud/vcd/utils.py

```diff
@@ -170,12 +167,16 @@ def vapp_to_dict(vapp, metadata=None):
                         value = '{:,} {}'.format(int(quantity), units).strip()
                 else:
                     value = '{}: {}'.format(
-                                        connection.get('{http://www.vmware.com/vcloud/v1.5}ipAddressingMode'),  # NOQA
-                                        connection.get('{http://www.vmware.com/vcloud/v1.5}ipAddress'))  # NOQA
+                        connection.get(
+                            '{http://www.vmware.com/vcloud/v1.5}ipAddressingMode'  # NOQA
```

Comment:
We should replace all these hard coded strings with the corresponding item in NSMAP in a future commit.

Reply:
will remove it in my current development branch